manifests/filter-by-terms/search/queue-subscriptions.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: search-queue-subscriptions
+  namespace: journal--filter-by-term
+spec:
+  interval: 1m0s
+  sourceRef:
+    kind: GitRepository
+    name: journal-team-deployment
+    namespace: flux-system
+  path: ./kustomizations/search/queue-subscriptions
+  prune: true
+  targetNamespace: journal--filter-by-term
+  dependsOn:
+  - name: search-queue
+  postBuild:
+    substituteFrom:
+    # search_queue_arn and search_queue_name are retrieved from the configmap
+    - kind: ConfigMap
+      name: search-queue-config
+    substitute:
+      # deployment vars
+      env: prod
+
+      # Which SNS topics should the queue be subscribed to?
+      bus_articles_topic_arn: arn:aws:sns:us-east-1:512686554592:bus-articles--filter-by-term
```

Comment:
Not sure these exist? That's not an environment that exists prior to this deployment, so nothing is sending notifications to those topics. So maybe this whole subscriptions are not needed?